packages/playground/src/utils/migration.ts

```diff
@@ -1,7 +1,52 @@
-import type { BaseModule } from "@threefold/grid_client/dist/es6/modules/base";
-import type { Contract } from "@threefold/tfchain_client";
+import type { Contract, ExtrinsicResult } from "@threefold/tfchain_client";
+
+import { BaseModule } from "../../../grid_client/dist/es6/modules/base";
```

Comment:
```suggestion
import { BaseModule } from "@threefold/grid_client/dist/es6/modules/base";
```

Reply:
When used it I got this error, maybe it's related to the package dist files?

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/57001890/bbe15a76-fa31-4d3c-bccc-91804e397bff)